Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6.8-alpine3.9
+RUN apk add --no-cache bash
+WORKDIR /app
+COPY setup.cfg setup.py README.md CHANGELOG.md ./
+COPY sceptre/ ./sceptre
+RUN python setup.py install
+WORKDIR /project
+CMD ["sceptre"]
```

Comment:
For containers like this I would suggest we go with ENTRYPOINT here. It means that executing the container looks like:

```
$ docker run --rm -v `pwd`:/project:ro -it cloudreach/sceptre:latest launch prod
```

rather than redundantly specifying sceptre as the cmd. This has always seemed the pattern for a single application container such as this.

Also people will quickly come to think of this as a stable interface so it's best to decide now rather than change it later.